opencti-platform/opencti-front/src/utils/hooks/useBuildReadableAttribute.tsx

```diff
@@ -0,0 +1,47 @@
+import { renderToString } from 'react-dom/server';
+import React from 'react';
+import { dateFormat } from '../Time';
+import { useBuildFilterKeysMapFromEntityType } from '../filters/filtersUtils';
+import type { WidgetColumn } from '../widget/widget';
+
+const buildStringAttribute = (val: unknown, isDateAttribute: boolean) => {
```

Comment:
Why `val` isn't just  a string instead of unknown? what else could it be?

Reply:
it can be an array of string, a integer...